RUN.md

```diff
@@ -140,13 +140,18 @@ For testing, we build and run locally, using the Signal staging network, and the
     ```
    
     Note: If you are having issues with psycopg2, you may want to make sure that Python 3.9.5 is installed and postgresql is installed.
+    If you've got old versions of the mobilecoin CLI and you're having trouble getting pip to install the desired hash, use
+    the pip args `--no-cache-dir` and `--force-reinstall`.
    
 1. Prepare the environment for Django. (Make sure you have obtained a [Google Maps Client Key](https://developers.google.com/maps/documentation/maps-static/get-api-key))
```

Comment:
General question...what is a Google Maps Client Key needed for?  I don't think I set this `GMAPS_CLIENT_KEY` envar when I was testing locally the other day.

Reply:
Oh, it's *not* needed at the moment. This is a thing for address checking for Item drops.